deploy/kessel-inventory-ephem-w-debezium.yaml

```diff
@@ -20,6 +20,8 @@ objects:
         consumer:
           bootstrap-servers: inventory-kafka-kafka-bootstrap:9092
           topic: outbox.event.kessel.tuples
+          read-after-write-enabled: true # false == off for all service providers
+          read-after-write-allowlist: [] # specify "*" to require all requests
```

Comment:
I don't know if `consumer` is the right spot for this config, since it's not actually being leveraged by the consumer. Perhaps we should add a `consistency` section. Thoughts?

Reply:
the consumer would need to know about whether r-a-w is enabled or not to send the proper notification but i could see the argument that its not just the consumer that cares about that setting.

Reply:
I'm not against adding a `consistency` section, and it may not be leveraged directly by the consumer, but I was thinking that if the `consumer` was removed we wouldn't have a reason to keep the `read-after-write` features themselves and so couple them together.

Reply:
I think my hang-up is that read-after-write is more of an API feature than a consumer feature. We could support read-after-write without the consumer we'd just need to devise a different solution for the pub sub (kafka or something)

Maybe it makes sense in the new `resources` section?  Like you mentioned in slack, this feature really is only applicable to resources.